src/libraries/pretest.proj

```diff
@@ -138,5 +138,20 @@
                              RootAttributes="@(FrameworkListRootAttribute)" />
   </Target>
 
+  <Target Name="ILLinkTrimOOBLibraries"
```

Comment:
What's the reason that you aren't doing this in src.proj where we also trim the inbox ones?

Reply:
Is it that you need a ready to use runtime pack?

Reply:
We do the shared framework on src.proj because we are manually invoking the linker and editing the command that we run, while for the OOBs we are instead following [the steps we are asking third party library devs to follow](https://github.com/dotnet/docs/blob/main/docs/core/deploying/prepare-libraries-for-trimming.md) but for that to work, we need to have the RuntimeList.xml file already generated in order to be able to build trimming.csproj against the produced shared framework, and that only happens after pretest.proj.

Reply:
>Is it that you need a ready to use runtime pack?

Yup, exactly

Reply:
Hmm does that mean we can't restore before the runtime pack is available? If yes, did we consider treating this as a test instead of the product build? I.e. sourcebuild likely doesn't care about this logic.